Cannot build using `make esp32` - missing `sdkconfig.h`

```
.../toit/third_party/esp-idf/components/newlib/platform_include/assert.h:20:10: fatal error: sdkconfig.h: No such file or directory
 #include <sdkconfig.h>
          ^~~~~~~~~~~~~
compilation terminated.
ninja: build stopped: subcommand failed.
make: *** [build/esp32/lib/libtoit_image.a] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -149,7 +149,7 @@ build/$(ESP32_CHIP)/toit.bin build/$(ESP32_CHIP)/toit.elf: build/$(ESP32_CHIP)/l
 	make -C toolchains/$(ESP32_CHIP)/
 	$(TOITVM_BIN) tools/inject_config.toit build/config.json build/$(ESP32_CHIP)/toit.bin
 
-build/$(ESP32_CHIP)/lib/libtoit_image.a: build/$(ESP32_CHIP)/$(ESP32_CHIP).image.s build/$(ESP32_CHIP)/CMakeCache.txt
+build/$(ESP32_CHIP)/lib/libtoit_image.a: build/$(ESP32_CHIP)/$(ESP32_CHIP).image.s build/$(ESP32_CHIP)/CMakeCache.txt  build/$(ESP32_CHIP)/include/sdkconfig.h
 	(cd build/$(ESP32_CHIP) && ninja toit_image)
 
 build/$(ESP32_CHIP)/$(ESP32_CHIP).image.s: build/$(ESP32_CHIP)/ build/snapshot $(TOITVM_BIN) $(SNAPSHOT_DIR)/snapshot_to_image.snapshot
@@ -163,7 +163,9 @@ build/$(ESP32_CHIP)/CMakeCache.txt: build/$(ESP32_CHIP)/
 
 build/$(ESP32_CHIP)/:
 	mkdir -p $@
-	make -C toolchains/$(ESP32_CHIP) -s "$(CURDIR)"/build/$(ESP32_CHIP)/include/sdkconfig.h
+
+build/$(ESP32_CHIP)/include/sdkconfig.h: build/$(ESP32_CHIP)/
+	make -C toolchains/$(ESP32_CHIP) -s "$(CURDIR)"/$@
 
 build/config.json:
 	echo '{"wifi": {"ssid": "$(ESP32_WIFI_SSID)", "password": "$(ESP32_WIFI_PASSWORD)"}}' > $@

